pkg/cache/application_test.go

```diff
@@ -177,6 +263,32 @@ func (ms *mockSchedulerAPI) ReloadConfiguration(rmID string) error {
 	return nil
 }
 
+type mockRecorder struct {
```

Comment:
Pls rename this struct to `MockedRecorder` and move it to a separate file and put it into https://github.com/apache/incubator-yunikorn-k8shim/tree/master/pkg/common/events package. So in the future we can easily reuse it.

Reply:
OK, done.